PlanPluginBridge/src/main/java/com/djrapitops/pluginbridge/plan/luckperms/LuckPermsData.java

```diff
@@ -0,0 +1,107 @@
+import com.djrapitops.plan.data.plugin.PluginData;
+import com.djrapitops.plan.utilities.html.icon.Color;
+import com.djrapitops.plan.utilities.html.icon.Family;
+import com.djrapitops.plan.utilities.html.icon.Icon;
+import com.djrapitops.plan.utilities.html.icon.Icons;
+import java.util.Collection;
+import java.util.List;
+import java.util.OptionalInt;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import me.lucko.luckperms.api.*;
+import me.lucko.luckperms.api.caching.MetaData;
+
+public class LuckPermsData extends PluginData {
+    private LuckPermsApi api;
+
+    public LuckPermsData(LuckPermsApi api) {
+        super(ContainerSize.THIRD, "LuckPerms");
+        setPluginIcon(Icon.called("exclamation-triangle").of(Family.SOLID).build());
+
+        this.api = api;
+    }
+
+    @Override
+    public InspectContainer getPlayerData(UUID uuid, InspectContainer inspectContainer) {
+        User user = api.getUser(uuid);
+
+        if (user == null) {
+            inspectContainer.addValue("Data unavailable", "Could not get user data");
+            return inspectContainer;
+        }
+
+        MetaData metaData = user.getCachedData().getMetaData(Contexts.allowAll());
+        String prefix = metaData.getPrefix();
+        String suffix = metaData.getSuffix();
+
+        inspectContainer.addValue(getWithIcon("Primary group", Icon.called("user-friends").of(Family.SOLID)), user.getPrimaryGroup());
+
+
+        inspectContainer.addValue(getWithIcon("Prefix", Icon.called("file-signature").of(Family.SOLID).of(Color.GREEN)), prefix != null ? prefix : "None");
+        inspectContainer.addValue(getWithIcon("Suffix", Icon.called("file-signature").of(Family.SOLID).of(Color.BLUE)),suffix != null ? suffix : "None");
+
+        TableContainer metaTable = new TableContainer(
+                getWithIcon("Meta", Icon.called("info-circle").of(Family.SOLID)),
+                getWithIcon("Value", Icon.called("file-alt").of(Family.SOLID))
+        );
+        metaData.getMeta().forEach((key, value) -> metaTable.addRow(key, value));
+        inspectContainer.addTable("Meta", metaTable);
+
+        List<String> groups = user.getPermissions().stream().filter(Node::isGroupNode).map(Node::getGroupName).collect(Collectors.toList());
+
+        TableContainer groupTable = new TableContainer(getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID)));
+        groups.forEach(groupTable::addRow);
+        inspectContainer.addTable("Groups", groupTable);
+
+        TableContainer trackTable = new TableContainer(
+                getWithIcon("Track", Icon.called("ellipsis-h").of(Family.SOLID)),
+                getWithIcon("Group", Icon.called("user-friends").of(Family.SOLID))
+        );
+        for (Track track : api.getTracks()) {
+            // reduce is used to get the last element
+            String currentGroup = api.getGroups().stream().map(this::getGroupDisplayName).filter(groups::contains).reduce((first, second) -> second).orElse("None");
```

Comment:
This line is long, I think a couple line breaks would do it justice.
Nice use of reduce!